buildpack/infrastructure/storage.py

```diff
@@ -134,6 +134,13 @@ def _get_s3_specific_config(vcap_services):
         if runtime.get_runtime_version() < 7.19:
             # Deprecated in 7.19
             config[config_prefix + "PerformDeleteFromStorage"] = False
+        elif runtime.get_runtime_version() >= (
+            MXVersion("9.6.11") or MXVersion("9.12")
```

Comment:
Not really familiar with how Python does it, but this rings a bit of an alarm bell to me - in other languages, this would be equal to comparing one value to whether one of the other values is true (which is always the case here).

Reply:
I've made it consistent with the way that we compare versions in the rest of the code now.